tests/green_mode/green_modeTest.py

```diff
@@ -0,0 +1,346 @@
+import os
+import unittest
+import yaml
+from copy import deepcopy
+from pathlib import Path
+
+from coala_quickstart.generation.SettingsClass import (
+    collect_bear_settings)
+from coala_quickstart.green_mode.Setting import (
+    find_max_of_setting, find_min_of_setting,)
+from coala_quickstart.green_mode.green_mode import (
+    initialize_project_data, generate_complete_filename_list,
+    run_quickstartbear, get_setting_type, get_kwargs,
+    check_bear_results, local_bear_test, global_bear_test,
+    bear_test_fun)
+from coala_quickstart.generation.Utilities import (
+    append_to_contents,
+    dump_yaml_to_file,
+    get_yaml_contents,
+    )
+from coala_quickstart.green_mode.QuickstartBear import (
+    QuickstartBear)
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange, SourcePosition
+from tests.test_bears.AllKindsOfSettingsDependentBear import (
+    AllKindsOfSettingsBaseBear)
+from tests.test_bears.TestGlobalBear import TestGlobalBear
+from tests.test_bears.TestLocalBear import TestLocalBear
+
+settings_key = 'green_mode_infinite_value_settings'
+
+
+class Test_green_mode(unittest.TestCase):
+
+    def test_get_yaml_contents(self):
+        project_data = 'example_.project_data.yaml'
+        full_path = str(Path(__file__).parent / project_data)
+        yaml_contents = get_yaml_contents(full_path)
+        test_yaml_contents = {'dir_structure':
```

Comment:
move `'dir_structure':` down to the next line, indented by four spaces from `test_y...` and then indent everything else accordingly from there.